sbc_compassion/models/correspondence_template.py

```diff
@@ -129,6 +129,7 @@ class CorrespondenceTemplate(models.Model):
         copy=True,
         readonly=False,
     )
+    website_published = fields.Boolean(default = True)
```

Comment:
This has nothing to do with the task?

Reply:
Yes, it has nothing to do.  I can do a commit to remove it.